internal/errorlint/errorlint.go

```diff
@@ -0,0 +1,140 @@
+// Copyright 2023 Outreach Corporation. All Rights Reserved.
+
+// Description: See package comment for this one file package.
+
+// Package errorlint contains the necessary logic for the error/log/trace linter.
+package errorlint
+
+import (
+	"go/ast"
+	"go/token"
+	"strconv"
+	"unicode"
+	"unicode/utf8"
+
+	"github.com/getoutreach/lintroller/internal/common"
+	"github.com/getoutreach/lintroller/internal/reporter"
+	"golang.org/x/tools/go/analysis"
+)
+
+// name defines the name for the errorlint linter.
+const name = "errorlint"
+
+// doc defines the help text for the error linter.
+const doc = `Ensures that each static message in error/trace/log/ is lowercase.
+
+A valid example is the following:
+
+	func foo() { errors.New("org not found in launchdarkly rule")}`
```

Comment:
It'd probably be a good idea to also provide an invalid example. I would just go ahead and drop the `func foo() {}` wrapper around the `errors.New` call - it's invalid syntax the way it sits anyways and the `errors.New` call alone will convey your point

Reply:
fixed